core/src/themes/base/high-contrast-dark.tokens.ts

```diff
@@ -0,0 +1,174 @@
+import { generateColorSteps, mix } from '../../utils/theme';
+import type { HighContrastDarkTheme } from '../themes.interfaces';
+
+import { darkPrimitiveColors as baseDarkPrimitiveColors } from './dark.tokens.primitives';
+
+const colors = {
+  primary: '#7cabff',
+  secondary: '#62bdff',
+  tertiary: '#b6b9f9',
+  success: '#4ada71',
+  warning: '#ffce31',
+  danger: '#fc9aa2',
+  light: '#222428',
+  medium: '#a8aab3',
+  dark: '#f4f5f8',
+  ...baseDarkPrimitiveColors,
+};
+
+export const highContrastDarkTheme: HighContrastDarkTheme = {
+  enabled: 'never',
+  color: {
+    primary: {
```

Comment:
We really create a function to generate these. All the themes are using the same structure and mix percentages.